### DISCUSSION TITLE
Draw 3D volume in the excel file with x,y,z,v

### DISCUSSION THREAD

**Discussion by subenyu** [pyvista/pyvista#1526] [Q&A] [answered]
Draw 3D volume in the excel file with x,y,z,v 
[11.xlsx](https://github.com/pyvista/pyvista/files/6837111/11.xlsx)

**Comment by akaszynski** [comment, marked as answer]
Hello,

We're going to need a little bit more help to get started with this.  Could you please:
- Output the data to a neutral format (e.g. CSV).
- Post code you've tried.

Thanks!
